docs/demo/index.rst

```diff
@@ -10,24 +10,20 @@ the left sidebar to see how various elements look on this theme.
     :maxdepth: 2
     :caption: Demo Documentation
 
-    structure
-    demo
+    kitchen-sink/index
+    theme-elements
     api
-    lists_tables
-    markdown
     example_pandas
-    mult_headers
-    subpages/index
-    subpages2/index
```

Comment:
For some reason I added two collapsible pages next to each other in https://github.com/pydata/pydata-sphinx-theme/pull/349 (the subpages2 in addition to subpages), but I don't remember if that had a practical reason for testing .. (I suppose we can always add it back when we discover something related to it)